vowpalwabbit/parse_args.cc

```diff
@@ -219,6 +219,7 @@ void parse_dictionary_argument(vw& all, const std::string& str)
   {
     fd = VW::io::open_file_reader(fname);
   }
+
```

Comment:
remove this added newline between try and catch

Reply:
Done.